docs/source/v1.10.md.inc

```diff
@@ -12,9 +12,11 @@
 
 [//]: # (- Whatever (#000 by @whoever))
 
-[//]: # (### :bug: Bug fixes)
+### :bug: Bug fixes
 
-[//]: # (- Whatever (#000 by @whoever))
+- When running the pipeline with [`mf_filter_chpi`][mne_bids_pipeline._config. mf_filter_chpi] enabled,
```

Comment:
Unless the space was added intentionally I assume it shouldn't be there. Will have to look at the website once it's built to see if it matters. (It may or may not error, I'll look by eye.)
```suggestion
- When running the pipeline with [`mf_filter_chpi`][mne_bids_pipeline._config.mf_filter_chpi] enabled,
```

Reply:
Indeed it's problematic

https://output.circle-artifacts.com/output/job/62c1acc3-c225-4dcb-bdf8-94b0b70b9328/artifacts/0/site/changes.html

<img width="751" alt="Screenshot 2024-07-30 at 7 16 17 PM" src="https://github.com/user-attachments/assets/d103f3de-799d-4679-9c52-f43f79bbbfd7">

@hoechenberger any way to make these links loudly rather than silently fail? I'm not so familiar with the `mkdocs` settings that might be the equivalent of `-nW` in sphinx (nitpicky + warning-as-error) that would have caught this

Reply:
I put in the space because I was copying the format of the first bugfix in the 1.9 changelog (which is also not rendering, now that I look at the site: https://mne.tools/mne-bids-pipeline/stable/changes.html#bug-fixes)

thought it was odd but assumed it was some fiddly requirement of mkdocs